src/main/java/org/broadinstitute/hellbender/transformers/SimpleRepeatMaskTransformer.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.hellbender.transformers;
+
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+/**
+ * Masks read bases with a supra-threshold number of A/T's or G/C's within a given window size.
+ */
+public class SimpleRepeatMaskTransformer implements ReadTransformer {
+    public static final long serialVersionUID = 1L;
+    private static final byte MASKED_BASE_QUAL = 2;
+
+    final int windowSize, threshAT, threshGC;
+
+    public SimpleRepeatMaskTransformer(final int threshAT, final int threshGC, final int windowSize) {
+        this.threshAT = threshAT;
```

Comment:
Maybe good to put in an assert that threshAT and threshGC are less than windowSize (if not, it indicates a typo or logic error).